From 6bd45f2e78f31bde335f7720e570a07331031110 Mon Sep 17 00:00:00 2001
Message-Id: <6bd45f2e78f31bde335f7720e570a07331031110.1648734973.git.geliang.tang@suse.com>
From: Liu xuzhi <liu.xuzhi@zte.com.cn>
Date: Thu, 11 Mar 2021 04:31:03 -0800
Subject: [PATCH] kernel/bpf/: Fix misspellings using codespell tool
Git-commit: 6bd45f2e78f31bde335f7720e570a07331031110
Patch-mainline: v5.13-rc1
References: bsn#127
Modified-by-SEL: No


A typo is found out by codespell tool in 34th lines of hashtab.c:

$ codespell ./kernel/bpf/
./hashtab.c:34 : differrent ==> different

Fix a typo found by codespell.

Signed-off-by: Liu xuzhi <liu.xuzhi@zte.com.cn>
Signed-off-by: Alexei Starovoitov <ast@kernel.org>
Link: https://lore.kernel.org/bpf/20210311123103.323589-1-liu.xuzhi@zte.com.cn
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 kernel/bpf/hashtab.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/bpf/hashtab.c b/kernel/bpf/hashtab.c
index 330d721dd2af..d7ebb12ffffc 100644
--- a/kernel/bpf/hashtab.c
+++ b/kernel/bpf/hashtab.c
@@ -31,7 +31,7 @@
 /*
  * The bucket lock has two protection scopes:
  *
- * 1) Serializing concurrent operations from BPF programs on differrent
+ * 1) Serializing concurrent operations from BPF programs on different
  *    CPUs
  *
  * 2) Serializing concurrent operations from BPF programs and sys_bpf()
-- 
2.34.1

